howso/client/base.py

```diff
@@ -849,8 +849,8 @@ def move_cases(
         precision: t.Optional[Precision] = None,
         preserve_session_data: bool = False,
         source_id: t.Optional[str] = None,
-        source_name_path: t.Optional[Collection[str]] = None,
-        target_name_path: t.Optional[Collection[str]] = None,
+        source_path: t.Optional[Collection[str]] = None,
+        target_path: t.Optional[Collection[str]] = None,
```

Comment:
Maybe we should make it clear in the docstring that one of id/path is required for both source and target